src/common.js

```diff
@@ -32,12 +32,18 @@ export const checkStatus = (response) => {
   return Promise.reject(error);
 };
 
-export const makeFetchArgs = (method, body) => {
+export const makeFetchArgs = (method, body, options) => {
   const init = {
     method: method,
     headers: { 'Accept': JSONTYPE, 'Content-Type': JSONTYPE }
   };
 
+  if (options) {
+    if (options.credentials) {
```

Comment:
I think you can combine these two if statements `if (options && options.credentials)`